hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -3431,15 +3431,27 @@
   <property>
     <description>
     When "yarn.node-labels.configuration-type" is configured with
-    "delegated-centralized", then periodically node labels are retrieved
-    from the node labels provider. This configuration is to define the
-    interval. If -1 is configured then node labels are retrieved from
-    provider only once for each node after it registers. Defaults to 30 mins.
+    "delegated-centralized", then periodically node labels of cluster
+    all nodes are retrieved from the node labels provider. This
+    configuration is to define the interval. If -1 is configured then
+    node labels are retrieved from provider only once for each node
+    after it registers. Defaults to 30 mins.
     </description>
     <name>yarn.resourcemanager.node-labels.provider.fetch-interval-ms</name>
     <value>1800000</value>
   </property>
 
+  <property>
+    <description>
+      When "yarn.node-labels.configuration-type" is configured with
+      "delegated-centralized", then periodically node labels from newly
+      registered nodes are retrieved from the node labels provider.
+      Defaults to 30 secs.
```

Comment:
Could you rephrase it as:
"When "yarn.node-labels.configuration-type" is configured with "delegated-centralized", then node labels of newly registered nodes are updated by periodically retrieving node labels from the provider. Defaults to 30 secs."